### ISSUE TITLE
[FEATURE] Use Converse API instead of InvokeModel API for Amazon Bedrock integrations

### ISSUE THREAD

**Issue by AndrewSerbin** [langchain4j/langchain4j#1789]
Currently, Langchain4j’s Amazon Bedrock integration utilizes the InvokeModel API. Amazon has introduced the Converse API, which provides a consistent interface that works with all models that support messages. https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_Converse.html

**Comment by langchain4j** [comment]
@AndrewSerbin sounds great! Would you like to implement this? 🙏

**Comment by herbert-beckman** [comment]
@langchain4j see my example in issue #1074. I propose a new architecture to implement the converse API or create a new package.

**Comment by langchain4j** [comment]
@herbert-beckman if converse API uses the same SDK, I would keep it in `langchain4j-bedrock` module.
We could create `BedrockChatModel` that could eventually be used instead of all `Bedrock{Model}ChatModel` classes.
WDYT?

**Comment by herbert-beckman** [comment]
I agree. However, some details still remain from the models, such as the `input schemas` and `inference configs`. I'm going to open a PR with the proposal.

**Comment by langchain4j** [comment]
@herbert-beckman awesome, thank you!

**Comment by langchain4j** [comment]
Hi @herbert-beckman, any news regarding this? Thanks!

**Comment by herbert-beckman** [comment]
Hello @langchain4j. No news. I haven't been able to focus on it yet.

**Comment by herbert-beckman** [comment]
@langchain4j I opened the draft PR. Please, if you can already suggest any changes, I'd appreciate it. Thank you!

**Comment by langchain4j** [comment]
@herbert-beckman great, thank you! ❤️  Looks good from the first glance, I will take a deeper look this week!
